src/classes/project.py

```diff
@@ -265,17 +268,22 @@ def _recurse_tree(self, bl_el=None, proj_el=None, parent: QStandardItem = None):
             new_proj_el.find('Path')
 
             layer_name = None
-            layer_path = new_proj_el.find('Path')
-            if layer_path is None:
-                self.settings.log("Could not find <Path> element on line {} of file: {}".format(new_proj_el.sourceline, self.business_logic_path), Qgis.Critical)
-                return
 
-            layer_uri = os.path.join(self.project_dir, new_proj_el.find('Path').text)
             # If this is a geopackage it's special
             if new_proj_el.getparent().tag == 'Layers':
-                layer_name = new_proj_el.find('Path').text
+                if self.project.attrib['{http://www.w3.org/2001/XMLSchema-instance}noNamespaceSchemaLocation'] == 'https://xml.riverscapes.xyz/Projects/XSD/V2/RiverscapesProject.xsd':

```

Comment:
can we set `self.version` on the project so that we don't need to have this logic in two places?

Also I think we should remove the hardcoded aspects of this url to the config file. I'll elaborate when we talk.